src/app/app-components/courses/courses-list/courses-list.component.html

```diff
@@ -1,8 +1,14 @@
-<div *ngFor="let course of courses">
-    <app-courses-item [courseInfo]="course" (deleteCourse)="onDeleteItem($event)">
-    </app-courses-item>
-</div>
+<ng-container *ngIf="courses && courses.length > 0">
```

Comment:
Later, take a look on Angular.io for how to use if/else statement in templates.

Reply:
Sure!

it's just that I'm used to avoiding using ngIfElse with the client I'm working for - I'm sorry!